package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@viamrobotics/prime",
   "version": "0.0.100",
   "license": "Apache-2.0",
-  "type": "commonjs",
+  "type": "module",
```

Comment:
The latest version of the vite plugin for svelte no longer supports commonJS, and luckily storybook 7 has no issues with using ES modules (unlike v6).